src/components/upload-queue/QueueItemTable.tsx

```diff
@@ -0,0 +1,69 @@
+import { Component, Suspense } from 'solid-js'
+import { For, Match, Switch } from 'solid-js'
+import { Transition, TransitionGroup } from 'solid-transition-group'
+
+import Icon from '~/components/material/Icon'
+import { UploadItem } from '~/types'
+
+import QueueItem from './QueueItem'
+import clsx from 'clsx'
+
+const animations = (slide: boolean) => {
```

Comment:
Animations look like something which could just be added in a separate PR later, where there is time to test them properly